src/components/basic/DlPanelContainer/DlPanelContainer.vue

```diff
@@ -518,7 +573,6 @@ export default defineComponent({
 
         & .content {
             overflow-x: hidden;
-            overflow-y: scroll;

```

Comment:
Is this ok to be removed for every place using the `DlPanelContainer` ?

Reply:
It wont affect , scroll is working fine for y axis in left panel ui